src/main.rs

```diff
@@ -266,7 +291,7 @@ fn default_url() -> &'static str {
 }
 
 fn default_footer() -> &'static str {
-    r#"<a href="https://creativecommons.org/licenses/by-nc-sa/4.0/">CC-BY_NC-SA</a> | Site generated with <a href="https://github.com/rochacbruno/marmite">Marmite</a>"#
+    r#"<a href=\"https://creativecommons.org/licenses/by-nc-sa/4.0/\">CC-BY_NC-SA</a> | Site generated with <a href=\"https://github.com/rochacbruno/marmite\">Marmite</a>"#
```

Comment:
A change to the default_footer() function uses a raw string literal in Rust, denoted by r#""#, to set the string value